redesign/js/app/DialogueEvaluation.js

```diff
@@ -88,14 +88,19 @@ $.extend( DialogueEvaluation.prototype, {
 	},
 
 	_getEditingAttribution: function() {
+		var change, creator, attribution;
 		if( this._getResult( 'editing', 'edited' ) !== 'true' ) {
 			return '';
 		}
 
-		return this._getResult( 'change', 'change' )
-			+ ' ' + Messages.t( 'evaluation.by' )
-			+ ' ' + this._getResult( 'creator', 'name' )
-			+ ', ';
+		change = this._getResult( 'change', 'change' );
+		creator = this._getResult( 'creator', 'name' );
+		attribution = change ? change : Messages.t( 'evaluation.edited' );
```

Comment:
It takes one line to tell that I am not that experienced in javascript. I've changed this line